features/addressing/managing_zones/editing_zone.feature

```diff
@@ -0,0 +1,63 @@
+@managing_zones
+Feature: Editing a zone
+    In order to change the membership of areas
+    As an Administrator
+    I want to be able to edit a zone
+
+    Background:
+        Given the store has country "France"
+        And the store also has country "Germany"
+        And the store has a province "Alabama" with code "AL"
+        And the store has a province "Arizona" with code "AZ"
+        And the store has a zone "North America" with code "NA"
+        And the store has a zone "South America" with code "SA"
+        And I am logged in as an administrator
+
+    @ui @javascript
+    Scenario: Removing a country from a zone
+        Given the store has a zone "European Union" with code "EU"
+        And it has the "France" country member
+        And it also has the "Germany" country member
+        And I want to modify the zone named "European Union"
+        When I remove the "Germany" country member
+        And I save my changes
+        Then I should be notified about successful edition
+        And this zone should have only the "France" country member
+
+    @ui @javascript
+    Scenario: Removing a province from a zone
+        Given the store has a zone "United States of America" with code "USA"
+        And it has the "Alabama" province member
+        And it also has the "Arizona" province member
+        And I want to modify the zone named "United States of America"
+        When I remove the "Arizona" province member
+        And I save my changes
+        Then I should be notified about successful edition
+        And this zone should have only the "Alabama" province member
+
+    @ui @javascript
+    Scenario: Removing a zone from a zone
```

Comment:
Yo  dawg, I herd you like zones, so I put a zone inside a zone, so you can delete a zone while you delete a zone.